src/react/components/pages/signIn/signInPage.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react"
+import { ISignIn } from "../../../../models/applicationState";
+import SignInForm from "./signInForm";
+import { Route, Redirect } from "react-router-dom";
+import ApiService from "../../../../services/apiService"
+import IAuthActions, * as authActions from "../../../../redux/actions/authActions";
+import { showError } from "../../../../redux/actions/appErrorActions";
+
+
+export interface ISignInPageProps extends React.Props<SignInPage> {
+    actions: IAuthActions;
+}
+
+export interface ISignInPageState {
+    signin: ISignIn;
+}
+
+export default class SignInPage extends React.Component<ISignInPageProps, ISignInPageState> {
+    constructor(props){
+        super(props);
+        this.state = {
+            signin: null,
+        };
+        this.onFormSubmit = this.onFormSubmit.bind(this);
+
+    }
+
+    private async onFormSubmit(signin: ISignIn) {
```

Comment:
you have async function, so you could use `await` syntax instead of `then` and `catch`